content/en/_index.md

```diff
@@ -33,7 +33,8 @@ descending order of priority:
 
 1. **Clarity**: The code's purpose and rationale is clear to the reader
 2. **Idiomaticity**: The code should be written using idiomatic,
-   community-driven coding practices
+   community-driven coding practices, except where these practices
```

Comment:
+1 
This is an important exception for Terraform code samples

Reply:
To differentiate from our Terraform blueprints (https://cloud.google.com/docs/terraform/blueprints/terraform-blueprints), Terraform resource samples at go/tf-samples aren't allowing abstraction via modules, global variables, loops, etc. These are community-driven coding practices for Terraform, but we don't allow them.

Reply:
Captured in b/271024053 for further discussion, so closing here.